docs/addons/external-dns.md

```diff
@@ -9,11 +9,10 @@ The add-on provides functionality to configure IAM policies and Kubernetes servi
 ```typescript
 import * as ssp from '@shapirov/cdk-eks-blueprint';
 
-readonly externalDns = new ssp.addons.ExternalDnsAddon({
-    hostedZone: new ssp.addons.LookupHostedZoneProvider(myHostedZoneName)
-});
-
-const addOns: Array<ClusterAddOn> = [ externalDns ];
+const hostedZoneName '...'
```

Comment:
minor: `hostedZoneName '...'` - on purpose to have a compile issue? why not `hostedZoneName = ...` or something similar.